src/cli/cmd/mod.rs

```diff
@@ -31,6 +32,7 @@ pub(crate) enum FhSubcommands {
     Add(add::AddSubcommand),
     Search(search::SearchSubcommand),
     List(list::ListSubcommand),
+    Completions(completions::CompletionsSubcommand),
```

Comment:
kubectl, docker, hugo, and a lot of others use `<tool> completion <shell>` as their format. I'd prefer to stick with that format and make this `fh completion` instead.